<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>622</width>
    <height>418</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="balance_label">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>591</width>
      <height>91</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="goal_label">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>7</width>
      <height>18</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="progress_label">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>7</width>
      <height>18</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="remaining_months_label">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>7</width>
      <height>18</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="amount_input">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>100</y>
      <width>571</width>
      <height>91</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="deposit_button">
    <property name="geometry">
     <rect>
      <x>333</x>
      <y>200</y>
      <width>261</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>Пополнить</string>
    </property>
   </widget>
   <widget class="QPushButton" name="withdraw_button">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>200</y>
      <width>301</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>Снять</string>
    </property>
   </widget>
   <widget class="QPushButton" name="view_history_button">
    <property name="geometry">
     <rect>
      <x>23</x>
      <y>320</y>
      <width>571</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>История</string>
    </property>
   </widget>
   <widget class="QPushButton" name="set_goal_button">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>240</y>
      <width>571</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>Установить цель</string>
    </property>
   </widget>
   <widget class="QPushButton" name="clear_history_button">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>280</y>
      <width>571</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>Стереть</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>622</width>
     <height>27</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
